### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by Riannony**

### CODE REVIEW

File: src/module7/homework/task1/User.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    public User(long id, String firstName, String lastName, String city, int balance) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.city = city;
+        this.balance = balance;
+    }
+
+    @Override
+    public int compareTo(User o) {
```
Comment (Riannony) [inline src/module7/homework/task1/User.java]: rename o -> user

File: src/module7/homework/task1/Order.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        if (currency != order.currency) return false;
+        if (itemName != null ? !itemName.equals(order.itemName) : order.itemName != null) return false;
+        if (shopIdintificator != null ? !shopIdintificator.equals(order.shopIdintificator) : order.shopIdintificator != null)
+            return false;
+        return user != null ? user.equals(order.user) : order.user == null;
+
+    }
+
+    @Override
+    public int compareTo(Order o) {
```
Comment (Riannony) [inline src/module7/homework/task1/Order.java]: rename o -> order

File: src/module7/homework/task2/MainForList.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+    private static List<Order> deleteDublicateFromList(List<Order> orders) {
+        Set<Order> resultList = new HashSet<>();
+        resultList.addAll(orders);
+        return new ArrayList<>(resultList);
+    }
+
+    private static List<Order> deleteOrdersWithLowestPrice(List<Order> orders) {
+        List<Order> resultList = new ArrayList<>();
+        Iterator<Order> iter = orders.iterator();
```
Comment (Riannony) [inline src/module7/homework/task2/MainForList.java]: iter - > iterator

File: src/module7/homework/task3/MainForSet.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            if (user.getLastName().contains(lastNameForCheck)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private static Set<Order> deleteOrdersWithUSD(Set<Order> orders) {
+        Set<Order> result = new TreeSet<>(orders);
+        Iterator<Order> iter = result.iterator();
```
Comment (Riannony) [inline src/module7/homework/task3/MainForSet.java]: rename iter

File: src/module7/homework/task1/User.java (hunk 1 above)
Comment (nataliamad) [reply src/module7/homework/task1/User.java]: done

File: src/module7/homework/task1/Order.java (hunk 2 above)
Comment (nataliamad) [reply src/module7/homework/task1/Order.java]: done

File: src/module7/homework/task2/MainForList.java (hunk 3 above)
Comment (nataliamad) [reply src/module7/homework/task2/MainForList.java]: done

File: src/module7/homework/task3/MainForSet.java (hunk 4 above)
Comment (nataliamad) [reply src/module7/homework/task3/MainForSet.java]: done

**Pull request merged**